addons/website/static/src/snippets/s_website_form/options.js

```diff
@@ -1216,6 +1223,22 @@ options.registry.WebsiteFieldEditor = FieldEditor.extend({
         }
         fieldEl.classList.remove('s_website_form_field_hidden_if', 'd-none');
     },
+    /**
+     * Generates a new label that is different from the others in the form.
+     *
+     * @param {String[]} otherFieldLabels - The other field labels of the form.
+     * @returns {String} A new label that is not present on the form.
+     */
+    _generateNewLabel(otherFieldLabels) {
+        const myFieldName = this.$target[0].querySelector(".s_website_form_input").name;
```

Comment:
Could this be a parameter of the function to avoid to do `this.$target[0].querySelector(".s_website_form_input").name` twice (here and in the cleanForSave) ?